build.gradle.kts

```
@@ -24,7 +24,7 @@ plugins {
 val ecommerceCommonsVersion = "1.32.0"
 
 // eCommerce commons library git ref (by default tag)
-val ecommerceCommonsGitRef = ecommerceCommonsVersion
+val ecommerceCommonsGitRef = "CHK-3996-domain-object-reduce-hierarchy"
```

TODO: to restore after dependent commons pr merge
```suggestion
val ecommerceCommonsGitRef = "ecommerceCommonsVersion"
```